src/main/java/StringCalculator.java

```diff
@@ -54,7 +44,6 @@ public int sum(String[] strings){
 
             int num = checkException(n);
 
-            //예외 처리
             if(sum > Integer.MAX_VALUE - num || sum < Integer.MIN_VALUE + num) throw new RuntimeException("int 범위를 벗어났습니다.");
```

Comment:
```suggestion
            if(sum > Integer.MAX_VALUE - num || sum < Integer.MIN_VALUE + num) {
                throw new RuntimeException("int 범위를 벗어났습니다.");
            }
```

이렇게 하면 가독성이 좀 더 좋을 것 같아요

Reply:
넵 수정하겠습니다!